specification/eventgrid/data-plane/Microsoft.Storage/stable/2018-01-01/Storage.json

```diff
@@ -325,6 +325,34 @@
           "description": "The type of blob.",
           "type": "string"
         },
+        "accessTier": {
+          "description": "The current tier of the blob.",
+          "type": "string",
+          "enum": [
+            "Hot",
+            "Cool",
+            "Cold",
+            "Archive"
+          ],
+          "x-ms-enum": {
+            "name": "AccessTier",
+            "modelAsString": true
+          }
+        },
+        "previousTier": {
```

Comment:
Should this be called "previousAccessTier" for consistency?

Reply:
tier and accessTier have become interchangeable and is well known by users, we have been using previousTier in the log output format of our team's project 'Changefeed' for some time now.

Reply:
Then should accessTier just be tier? It seems they should be consistent.

Reply:
Regarding accessTier, we used the naming convention from the Azure documentation like here - https://learn.microsoft.com/en-us/rest/api/storageservices/list-blobs.

Regarding previousTier, we wanted to ensure consistency with the way this field is currently presented in the longer log-form output (Changefeed) of this service. In it, we have been publishing the field as previousTier, so we wanted the field in Change Notifications (which we send through Event Grid) to be consistent with this, hence the naming as previousTier here.

Reply:
We would prefer to name it previousAccessTier as it makes the meaning more obvious. The fact that it was named previousTier in change feed doesn't really change the fact that previousAccessTier is a more descriptive name. If for some reason, we can't change the tsp to use this name, we can customize it in the client libraries. It would be easier if we could just update the tsp.